packages/images-plugin/test/unit/finder.spec.js

```diff
@@ -2,8 +2,8 @@ import _ from 'lodash';
 import {imageUtil} from 'appium/support';
 import {BaseDriver} from 'appium/driver';
 import {ImageElementPlugin, IMAGE_STRATEGY} from '../../lib/plugin';
-import ImageElementFinder from '../../lib/finder';
-import ImageElement from '../../lib/image-element';
+import {ImageElementFinder, W3C_ELEMENT_KEY} from '../../lib/finder';
```

Comment:
should W3C_ELEMENT_KEY be imported from `@appium/support`?

Reply:
yeah I can do that